src/components/RecommendationTool.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import RecommendationHeader from './RecommendationHeader';
+import PropTypes from "prop-types";
+import DropdownSelect from './dropdown';
+
+const root = {
+    marginTop: 125
+}
+
+const descriptionText = {
+    textAlign:"center", 
+    marginTop: 25,
+    marginBottom: 25
+}
+
+const helpText = {
+    marginTop: -15,
+    marginBottom: 25,
+    textAlign: "center",
+    fontStyle: "italic"
+}
+
+const inputRoot = {
+    margin: "auto",
+    width: "max-content"
+}
+
+const nextButton = {
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+    "position": "absolute",
+    "width": "64px",
+    "height": "39px",
+    "right": "40px",
+    "bottom": "30px",
+    "background": "#3DBEFF",
+    "borderRadius": "100px",
+    "cursor": "pointer"
+}
+
+const previousButton = {
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+    "position": "absolute",
+    "width": "94px",
+    "height": "39px",
+    "left": "40px",
+    "bottom": "30px",
+    "background": "#3DBEFF",
+    "borderRadius": "100px",
+    "cursor": "pointer"
+}
+
+const buttonText = {
+    "fontFamily": "Roboto",
+    "fontStyle": "normal",
+    "fontWeight": "bold",
+    "fontSize": "16px",
+    "lineHeight": "19px",
+    "color": "#FFFFFF"
+}
```

Comment:
This and the "Recommendation Tool" button in RecommendationModal.js are the sort of situations I had in mind for a reusable ComparisonButton. It'd need a prop for turning off the chevron in the "Previous" and "Next" cases and a prop for styling it blue instead of an outline, but then there wouldn't be any need to hardcode widths and heights. Also, it'd make it easier to do an accessibility pass in the future if there are fewer button implementations to review.